security/src/main/java/no/difi/meldingsutveksling/auth/IdportenOidcTokenResponse.java

```diff
@@ -0,0 +1,49 @@
+package no.difi.meldingsutveksling.auth;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.google.common.base.MoreObjects;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class IdportenOidcTokenResponse {
+
+    private String accessToken;
+    private Integer expiresIn;
+    private String scope;
+
+    public IdportenOidcTokenResponse() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1186)